### PR TITLE
[Refactor] Support creating messages notifications

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

Created isMessageSilenced function for the `ZMConversation`. These changes are necessary to manage update events.

### CODE REVIEW

File: Source/Model/Conversation/ZMConversation+Mute.swift [hunk 1]
```diff
@@ -158,3 +158,35 @@ extension ZMConversationMessage {
     }
     
 }
+
+
+public extension ZMConversation {
+    func isMessageSilenced(_ message: GenericMessage, from senderID: UUID, in moc: NSManagedObjectContext) -> Bool {
```
Comment (typfel) [inline Source/Model/Conversation/ZMConversation+Mute.swift:164]: We can retrieve the `managedObjectContext` from the `ZMConversation` and remove `moc` argument. This also has the advantage that we can supply the wrong `NSManagedObjectContext` by mistake.

Comment (typfel) [inline Source/Model/Conversation/ZMConversation+Mute.swift:164]: We should modify the existing `isSilenced` method on `ZMConversationMessage` to use this method.

Comment (KaterinaWire) [reply Source/Model/Conversation/ZMConversation+Mute.swift]: Fixed

Comment (KaterinaWire) [reply Source/Model/Conversation/ZMConversation+Mute.swift]: It was done for the EAR and merged to this branch

**Pull request closed without merging**